src/unit_tests/wrappers/windows/processthreadsapi_wrappers.c

```diff
@@ -43,3 +43,12 @@ BOOL wrap_OpenProcessToken(__UNUSED_PARAM (HANDLE  ProcessHandle),
     *TokenHandle = mock_type(HANDLE);
     return mock();
 }
+
+/**
+ * @brief This function load the expect and will_return calls for the wrapper of SetThreadPriority
```

Comment:
This function **loads**. Also, this doxygen comment should be in the _.h_ file instead.